experimental/trusted_shuffler/trusted_shuffler/src/tests.rs

```diff
@@ -0,0 +1,100 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::TrustedShuffler;
+use futures::future::join_all;
+use std::{sync::Arc, time::Duration};
+use tokio::time::timeout;
+
+const TEST_ANONYMITY_VALUE: usize = 10;
+const TEST_TIMEOUT_MILLISECONDS: u64 = 100;
+
+async fn send_request(request: Vec<u8>) -> Vec<u8> {
+    request
```

Comment:
Maybe return something slightly different for the sake of the example, e.g. add an element or something. Also add a comment that this is just used as an example but in practice this function would talk to a backend etc.